gateway/src/service.rs

```diff
@@ -323,6 +327,17 @@ impl GatewayService {
         Ok(iter)
     }
 
+    /// The number of projects that are currently in the ready state
+    pub async fn count_ready_projects(&self) -> Result<u32, Error> {
```

Comment:
Nit: We have over 5k projects and the state is in JSON, is this query noticeably expensive to run? We can trace it, right?

Reply:
I already ran it in prod and it is instantaneous

Reply:
Cool!